jemalloc-sys/build.rs

```diff
@@ -188,8 +188,9 @@ fn main() {
     }
 
     if let Ok(malloc_conf_opts) = read_and_watch_env("JEMALLOC_SYS_WITH_MALLOC_CONF") {
-        malloc_conf += &format!(
-            "{}{}",
+        malloc_conf = format!(
```

Comment:
I think the expected fix for the warning should be
```
malloc_conf.push_str(xxx);
```

Reply:
Done.